include/deal.II/grid/tria.h

```diff
@@ -946,53 +945,60 @@ namespace internal
  * been called.
  *
  *
- * <h3>%Boundary approximation</h3>
+ * <h3>Describing curved geometries</h3>
  *
- * You can specify a class describing the boundary for each boundary component.
- * If a new vertex is created on a side or face at the boundary, this class is
- * used to compute where it will be placed. The manifold indicator of the face
- * will be used to determine the proper component. See Manifold for the details.
- * Usage with the Triangulation object is then like this (let @p Ball be a
- * class derived from Manifold<tt><2></tt>):
+ * deal.II implements all geometries (curved and otherwise) with classes
+ * inheriting from Manifold; see the documentation of Manifold, step-49, or
+ * the
+ * @ref manifold
+ * module for examples and a complete description of the algorithms. By
+ * default, all cells in a Triangulation have a flat geometry, meaning that
+ * all lines in the Triangulation are assumed to be straight. If a cell has a
+ * manifold_id that is not equal to zero then the Triangulation uses the
+ * associated Manifold object for computations on that cell (e.g., cell
+ * refinement). Here is a quick example that sets up a polar grid:
  *
- *   @code
- *     int main ()
- *     {
- *       Triangulation<2> tria;
- *                                        // set the boundary function
- *                                        // for all boundaries with
- *                                        // boundary indicator 0
- *       Ball ball;
- *       tria.set_manifold (0, ball);
- *
- *   // read some coarse grid
- *
- *
- *   Triangulation<2>::active_cell_iterator cell, endc;
- *   for (int i=0; i<8; ++i)
+ * @code
+ * int main ()
+ * {
+ *   PolarManifold<2> polar_manifold;
+ *   const types::manifold_id polar_id = 3;
+ *   Triangulation<2> triangulation;
+ *   GridGenerator::hyper_ball(triangulation);
+ *   triangulation.set_all_manifold_ids_on_boundary(polar_id);
+ *   triangulation.set_manifold(polar_id, polar_manifold);
+ *   for (int i=0; i<4; ++i)
```

Comment:
make this `unsigned int` while you're there. I know it's preexisting, but `unsigned` is just our usual style these days.